app/src/common/shared/com/igalia/wolvic/ui/adapters/Bookmark.java

```diff
@@ -32,7 +32,8 @@ public Bookmark(@NonNull BookmarkNode node, int level, boolean isExpanded) {
         mTitle = node.getTitle();
         mURL = node.getUrl();
         mGuid = node.getGuid();
-        mPosition = node.getPosition() != null ? node.getPosition() : 0;
+        mPosition = 0;
+        // mPosition = node.getPosition() != null ? node.getPosition() : 0;
```

Comment:
This looks suspicious to say the least.

Also we should not keep commented code.

Reply:
The position-related methods seem to get removed in the update, but I'm unsure if there are other ways to get the position. This one does need further investigation.